tests/cephfs/cephfs_utilsV1.py

```diff
@@ -968,6 +981,93 @@ def get_all_subvolumes(self, client, fs_list):
             subvolumes.extend([i["name"] for i in all_sub_info])
         return subvolumes
 
+    @function_execution_time
+    def create_nfs(self, client, nfs_cluster_name, validate=True, **kwargs):
+        """
+
+        Args:
+            client:
+            nfs_cluster_name:
+            validate:
+            **kwargs:
+
+        Returns:
+
+        """
+        nfs_cmd = f"ceph nfs cluster create {nfs_cluster_name}"
+        if kwargs.get("placement"):
+            nfs_cmd += f" --placement '{kwargs.get('placement')}'"
+
+        cmd_out, cmd_rc = client.exec_command(
+            sudo=True, cmd=nfs_cmd, check_ec=kwargs.get("check_ec", True)
+        )
+        if validate:
+            out, rc = client.exec_command(sudo=True, cmd="ceph nfs cluster ls")
+            nfscluster_ls = out.split("\n")
+            if nfs_cluster_name not in nfscluster_ls:
+                raise CommandFailed(f"Creation of filesystem: {nfs_cluster_name} failed")
```

Comment:
can we reword the failure to : creation of nfs cluster failed instead of filesystem?